utils/db.js

```diff
@@ -0,0 +1,21 @@
+import mysql from 'mysql';
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+const con = mysql.createConnection({
```

Comment:
create a config file, have this there and just import the config object from the config file and use it here..